modules/tinymce/changelog.txt

```diff
@@ -1,31 +1,31 @@
 Version 5.5.0 (TBD)
     Added a TypeScript declaration file to the bundle output for TinyMCE core #TINY-3785
-    Added new `table_column_resizing` setting to control how the table is affected when resizing columns using the resize bars #TINY-6001
-    Added ability to load TinyMCE in inline mode within a ShadowRoot #TINY-6128
+    Added new `table_column_resizing` setting to control how columns are resized when resizing tables using the resize bars #TINY-6001
+    Added the ability to load TinyMCE in inline mode within a ShadowRoot #TINY-6128
     Added the ability to remove images on a failed upload using the `images_upload_handler` failure callback #TINY-6011
     Added `hasPlugin` function to the editor API to determine if a plugin exists or not #TINY-766
```

Comment:
I wonder if we should use a better word than `exists`... Not sure what though tbh. This is an existing line though, Idk if we want to change it?

Reply:
loaded?

Reply:
We don't want loaded, as it doesn't check that by default. It'll just check if it's configured/enabled

Reply:
```suggestion
    Added `hasPlugin` function to the editor API to determine if a plugin is present in the editor configuration #TINY-766
```

Reply:
It does multiple things (it takes a boolean flag to check if it's just configured or configured and loaded), so this is now maybe misleading? This is probably still better than the original, so I'm okay with it.